include/cron.php

```diff
@@ -106,8 +107,8 @@ function cron_run(&$argv, &$argc){
 	// Check every conversation
 	ostatus::check_conversations(false);
 
-	// Set the gcontact-id in the item table if missing
-	item_set_gcontact();
+	// Do post update functions
```

Comment:
Question is: What can I say more? This is simply a call to a function that will call pending post update functions.